packages/teleport/src/Kubes/useKubes.ts

```diff
@@ -21,15 +21,18 @@ import TeleportContext from 'teleport/teleportContext';
 import useStickyClusterId from 'teleport/useStickyClusterId';
 
 export default function useKubes(ctx: TeleportContext) {
-  const { clusterId } = useStickyClusterId();
+  const { clusterId, isLeafCluster } = useStickyClusterId();
+  const user = ctx.storeUser.state.username;
+  const isEnterprise = ctx.isEnterprise;
+  const canCreate = ctx.storeUser.getTokenAccess().create;
   const { run, attempt } = useAttempt('processing');
   const [kubes, setKubes] = useState([] as Kube[]);
 
   useEffect(() => {
     run(() => ctx.kubeService.fetchKubernetes(clusterId).then(setKubes));
   }, [clusterId]);
 
-  return { kubes, attempt };
+  return { kubes, attempt, user, isEnterprise, isLeafCluster, canCreate };
```

Comment:
i'd create the `showButton` flag here, and export it, since we dont need `isEnterprise`, `isLeafCluster`, `canCreate` anywhere else